src/topology/category/Profinite/as_limit.lean

```diff
@@ -40,7 +40,7 @@ variables (X : Profinite.{u})
 
 /-- The functor `discrete_quotient X ⥤ Fintype` whose limit is isomorphic to `X`. -/
 def fintype_diagram : discrete_quotient X ⥤ Fintype :=
-{ obj := λ S, Fintype.of S,
+{ obj := λ S, by haveI := fintype.of_finite S; exact Fintype.of S,
```

Comment:
What's the effect of this change?

Reply:
I changed an instance from `noncomputable .. fintype` to `finite` elsewhere, so I have to use `fintype.of_finite` here.